src/CounterIndex.js

```diff
@@ -14,7 +14,12 @@ class CounterIndex {
   updateIndex(oplog) {
     if(this._index) {
       const createCounter = e => Counter.from(e.payload.value)
-      const mergeToIndex = e => this._index.merge(e)
+      const mergeToIndex = e => {
+        const other = new PNCounter(e.uid,
```

Comment:
```createCounter``` should return a PNCounter, is there a reason to not keep it as ```this._index.merge(e)```?

Reply:
I don't fully understand why, but when I do I end up with this error:
```
TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at GCounter.merge (/Users/jerome.bernard/Documents/Personnel/Development/orbit-db-counterstore/node_modules/crdts/src/G-Counter.js:39:12)
    at PNCounter.merge (/Users/jerome.bernard/Documents/Personnel/Development/orbit-db-counterstore/node_modules/crdts/src/PN-Counter.js:25:12)
    at mergeToIndex (/Users/jerome.bernard/Documents/Personnel/Development/orbit-db-counterstore/src/CounterIndex.js:17:45)
    at Array.forEach (<anonymous>)
    at CounterIndex.updateIndex (/Users/jerome.bernard/Documents/Personnel/Development/orbit-db-counterstore/src/CounterIndex.js:20:10)
    at CounterStore._updateIndex (/Users/jerome.bernard/Documents/Personnel/Development/orbit-db-counterstore/node_modules/orbit-db-store/src/Store.js:415:23)
    at Replicator.onLoadCompleted (/Users/jerome.bernard/Documents/Personnel/Development/orbit-db-counterstore/node_modules/orbit-db-store/src/Store.js:108:22)
```

I remember having seen at one point some mismatch in ```counters``` vs ```_counters```, but I'm not sure if it's for that specific change.

Anyway, the ```opens a saved counter``` test fails when reverting this change.

Reply:
Yes you're right it was due to ```toJSON``` method not serializing the counters correctly. This should be fixed now ([here](https://github.com/orbitdb/crdts/blob/31f5e8b17acea24cfd8aa2ab157935c724137b7b/src/PN-Counter.js#L32)) and we can leave the line as ```this._index.merge(e)```